docs/source/getting_started/active_learning.rst

```diff
@@ -100,11 +100,14 @@ Let's configure the sampling request and request an initial selection next:
 
    # we want an initial pool of 100 images
    config = SamplerConfig(n_samples=100, method=SamplingMethod.CORESET, name='initial-selection')
-   initial_selection = al_agent.query(config)
+   initial_selection, added_set = al_agent.query(config)
```

Comment:
I don't like this example here. Would not use 100 for both. This just introduces confusion. Maybe start with 200 images and then add another 100. I also don't get the difference between initial selection and added set here

Reply:
There is no difference in the initial step which is a little inelegant. Should we just go back to having a single return value? @MalteEbner thoughts?

Reply:
Why should I get a tuple? That does not really make sense to me. I query --> want to get back what I specified in the query

Reply:
I would go to have no return value at all, see my comments below.